app/schemas/astrology.py

```
@@ -15,6 +15,44 @@ class APIResponse(BaseModel):
+
+    @validator('birth_date')
+    def validate_birth_date(cls, v):
+        """Valida formato da data."""
+        try:
+            day, month, year = v.split('/')
+            if len(day) != 2 or len(month) != 2 or len(year) != 4:
+                raise ValueError()
+            day, month, year = int(day), int(month), int(year)
+            if not (1 <= day <= 31 and 1 <= month <= 12 and 1900 <= year <= 2100):
+                raise ValueError()
+            return v
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)